bindata/oauth-openshift/deployment.yaml

```diff
@@ -102,6 +103,16 @@ spec:
             periodSeconds: 10
             successThreshold: 1
             failureThreshold: 3
+          lifecycle:
+            # delay shutdown by 25s to ensure existing connections are drained
+            # * 5s for endpoint propagation on delete
+            # * 5s for route reload
+            # * 15s for the longest running request to finish
```

Comment:
When the pod is deleted, its endpoint is removed and we give it 5s to propagate. Existing connections keep serving, especially http/2 and keepalive ones. *New* requests will keep coming all these 25s, until the process gets a signal and it closes the connection.

Afaik we disallow http/2. What about keepalive, i.e. http connection reuse?

Does the CI monitor test force new connections for each request?